config/routes.rb

```diff
@@ -7,7 +7,7 @@
 
   namespace :api, defaults: { format: :json } do
     get '/v1/metrics/', to: "metrics#index"
-    get '/v1/metrics/:metric/*base_path/time-series', to: "metrics#time_series"
+    get '/v1/metrics/*base_path/time-series', to: "metrics#time_series"
     get '/v1/metrics/:metric/*base_path', to: "metrics#summary"
```

Comment:
As per the [2nd acceptance criteria](https://trello.com/c/vClxSNYC/617-3-update-api-to-support-multiple-metrics-per-basepath-epic-2) wouldn't we need to do the same with the aggregations?